kube/deploy.sh

```diff
@@ -2,7 +2,7 @@
 set -euo pipefail # make bash quit if something weird happens
 
 export KUBE_NAMESPACE=${ENVIRONMENT}
-export REPLICAS="1"
+export VERSION=${VERSION:-latest}
 export CA_URL="https://raw.githubusercontent.com/UKHomeOffice/acp-ca/master/acp-notprod.crt"
 
 export KUBE_CERTIFICATE_AUTHORITY=/tmp/cert.crt
```

Comment:
We can just pass in the URL here instead of doing the weird wget thing, turns out

Reply:
I initially did that, but with the SSL issue I rolled back incase I implemented it wrong. This has been changed...